src/Metrics.Extensions/src/Public/Microsoft.ApplicationInsights.Metrics/MetricSeriesConfigurationForAccumulator.cs

```diff
@@ -0,0 +1,200 @@
+﻿using System;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+
+using Microsoft.ApplicationInsights.Metrics.Extensibility;
+using Microsoft.ApplicationInsights.Metrics.Extensions;
+
+namespace Microsoft.ApplicationInsights.Metrics
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public class MetricSeriesConfigurationForAccumulator : IMetricSeriesConfiguration
+    {
+        private readonly bool _restrictToUInt32Values;
+        private readonly int _hashCode;
+
+        static MetricSeriesConfigurationForAccumulator()
+        {
+            MetricAggregateToTelemetryPipelineConverters.Registry.Add(
```

Comment:
this is super long line. Why the next line starts with to many "tabs" instead of a single "tab" indent

Reply:
Reduced indent in the next iteration.